uk.co.compucorp.civicrm.hrcore/CRM/HRCore/CMSData/UserMailNotifier/Drupal.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use CRM_HRCore_CMSData_UserMailNotifierInterface as UserMailNotifierInterface;
+
+/**
+ * Class CRM_HRCore_CMSData_UserMailNotifier_Drupal
+ */
+class CRM_HRCore_CMSData_UserMailNotifier_Drupal implements UserMailNotifierInterface {
+
+  /**
+   * @var stdClass
+   */
+  private $user;
+
+  /**
+   * CRM_HRCore_CMSData_UserMailNotifier_Drupal constructor.
+   *
+   * @param array $contactData
+   */
+  public function __construct($contactData) {
+    $this->user = user_load($contactData['cmsId']);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function sendWelcomeEmail() {
+   return _user_mail_notify('register_admin_created', $this->user);
```

Comment:
I'm not sure if we should be calling this function. Usually, functions starting with an underscore are considered private. Isn't there any other alternatives? Maybe you could check with a Drupal dev if it's ok to call this function and/or if there are alternatives